modules/dumbCommandModule/package.json

```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
+  "dependencies": {
+    "underscore": "latest"
```

Comment:
You shouldn't manually add dependencies in general; `npm install --save` will save a fairly sane semver selector. I'm not worried about underscore breaking though, so again no big deal.